src/config.rs

```diff
@@ -113,6 +114,27 @@ pub(crate) struct Config {
     /// Should be a org/repo code, e.g., rust-lang/rust.
     pub(crate) rustc_tag_repository: Option<String>,
 
+    /// Where to publish new blog PRs.
+    ///
+    /// We create a new PR announcing releases in this repository; currently we
+    /// don't automatically merge it (but that might change in the future).
+    ///
+    /// Should be a org/repo code, e.g., rust-lang/blog.rust-lang.org.
+    pub(crate) blog_repository: Option<String>,
+
+    /// The expected release date, for the blog post announcing dev-static
+    /// releases. This is trimmed & inserted verbatim into the blog post, format
+    /// is up to the user to decide.
+    pub(crate) scheduled_release_date: Option<String>,
+
+    /// Verbatim URL for insertion into the pre-release blog post. Not otherwise used.
+    pub(crate) release_notes_url: Option<String>,
```

Comment:
I would instead ask for the `YYYY-MM-DD` date of the release, and calculate both "pretty" values from them:

* Release date can be calculated with chrono's `.format("%B %d")` (for example "July 10"). It doesn't include the `st`/`rd`/`th` but oh well.

* Release notes url is:

  ```
  https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-{version_number_without_dots}-{date}
  ```

Reply:
Release notes haven't always landed on master, so we'd probably point to stable here (and edit any leftovers after post is made, I guess).

Otherwise this seems okay as a strategy.